### ISSUE TITLE
Notifier Not Opening

### ISSUE THREAD

**Issue by ghost** [Laumania/RobloxPlayerNotifier#2]
Hello, i have a problem were i have the notifier open and pinned to taskbar, i close it, then try to open it again and my cursor has a blue loading symbol next to it and then the loading symbol disappears and the app doesn't open, is there a fix? Or do i have to keep the app open all the time.

**Comment by ghost** [comment]
Also, how does it notify me? I had sombody go online and there was no sound, popup, nothing in the notification center, nothing at all. So if you know how to make it popup or make a noise, please tell me.

**Comment by Laumania** [comment]
Hi!

I have updated the Readme with a little more detailed guide on how to do this - check it out and let me know if you get it working:
https://github.com/Laumania/RobloxPlayerNotifier

**Comment by ghost** [comment]
I still have issues with this app, there is only 3 so the updated readme file helped. The 1st problem is whenever i close it and its pinned to taskbar, i cant open it again, this is a problem because the app doesn't have live updates and it only updates when you open it, i joined a game with my name in the notifier and it just said i was online, so i closed and opened it and it said i was in a game, it made the noise, but there was no notification on the bottom left.

TL;DR Cant open and close app when its pinned to taskbar
          Isnt live updates so i have to open and close it manually for it to update
          Was in a game, no notification on the bottom left (There was a sound so that's good enough, just telling you its not 
          showing)
(username is 1irx) btw

**Comment by Laumania** [comment]
It updates every 120 seconds, so it do live update, but only every 120 sec. You can lower it in the config if you want to like 10 or so. 

I can't say why you can't open it from your pinned taskbar...should work fine? But it's important that you have unzipped the whole zip folder first?

The notification is not visual, it's a sound/music playing.

**Comment by ghost** [comment]
Well, i use winRAR so that could be the issue why it doesnt open again, but probaly not. I just need to know what line it is to change it from 120 to 10, and im good to go. I cannot thank you enough for how helpful this app is and now i can actually find my favorite youtubers once i change the auto update time.
Also, i found a bug in the code were i change the names, it says "version="1.0" even though i downloaded version 1.1, should i change it manually to 1.1 or keep it at 1.0?

**Comment by Laumania** [comment]
Great to hear.

Try just use the build-in "Extract all..." that comes with Windows 10. Just right click on the zip and "Extract all..." then you get a folder with the same name and THAT's the place you need to make your changes and start the app from.

:)

**Comment by ghost** [comment]
LETS GOO!! You do not know how happy i am right now!!!! I uninstalled winRAR and used it normally, it does it in 20 seconds and it makes the noise, AND i can even open and close it!!! I am so happy right now and i cannot ever thank you enough for how good this is!!!! No problems whatsoever :D

**Comment by Laumania** [comment]
Im happy you got it all working :)
